boa/src/builtins/object/internal_methods.rs

```diff
@@ -57,7 +46,8 @@ pub trait ObjectInternalMethods {
     ///  - [ECMAScript reference][spec]
     ///
     /// [spec]: https://tc39.es/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-isextensible
-    fn is_extensible(&self) -> bool {
+    #[inline]
+    pub fn is_extensible(&self) -> bool {
```

Comment:
Hmmm do we need this to be public outside of the `boa` crate? Same with other functions here. Maybe this is more in line with #445, though.

Reply:
Should I make it `pub(crate)`?

Reply:
Hmmm let's leave it like this for now and re-visit it when we revamp the API.

Reply:
Yeah i don't think we need to change this right now